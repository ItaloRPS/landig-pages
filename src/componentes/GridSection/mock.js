export default {
    background:false,
    title:'My grid',
    description:'Atque doloribud nobis laudantium esse ut,dolor sit amet consectetur adipisicing elit. Id repudiandae non impedit asperiores sapiente! Consequatur accusamus mollitia error ratione harum dolores enim hic, doloremque labore inventore',
    grid:[
        {
            title:'Teste 1',
            description: 'Lorem ipsum, dolor sit amet consectetur adipisicing elit. Id repudiandae non impedit asperiores sapiente! Consequatur accusamus mollitia error ratione harum dolores enim hic, doloremque labore inventore provident dolor dolorem at!'
            
        },  
        {
            title:'Teste 2',
            description: 'Lorem ipsum, dolor sit amet consectetur adipisicing elit. Id repudiandae non impedit asperiores sapiente! Consequatur accusamus mollitia error ratione harum dolores enim hic, doloremque labore inventore provident dolor dolorem at!'
            
        },
        {
            title:'Teste 3',
            description: 'Lorem ipsum, dolor sit amet consectetur adipisicing elit. Id repudiandae non impedit asperiores sapiente! Consequatur accusamus mollitia error ratione harum dolores enim hic, doloremque labore inventore provident dolor dolorem at!'
            
        }

    ]
}