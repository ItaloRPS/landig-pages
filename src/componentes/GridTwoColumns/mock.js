export default {
    title:'Grid Two Columns',
    text: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Sit unde debitis vitae nesciunt praesentium ullam nulla saepe deleniti! Recusandae alias, praesentium numquam explicabo rerum esse vel? Explicabo, delectus. Perspiciatis, cupiditate.',
    srcImg: 'assets/imagens/javascript.svg'
}