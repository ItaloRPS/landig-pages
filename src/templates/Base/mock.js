import linksMock from '../../componentes/NavLinks/mock'
import gridMock from '../../componentes/GridSection/mock'
import {GridSection} from '../../componentes/GridSection'

export const mockBase = {
    children:(
        <>
            <GridSection {...gridMock}/>
            <GridSection {...gridMock} background/>
            <GridSection {...gridMock}/>
            <GridSection {...gridMock} background/>
        </>
    ),
        links:linksMock,
        logoData:{
            text:'Logo',
            link:'#'
        },
        footerHtml:'<p> Teste Footer </p>'
    
}
