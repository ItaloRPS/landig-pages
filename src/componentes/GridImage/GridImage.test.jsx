import {renderTheme} from '../../styles/render-theme'
import {GridImage} from '.'

import mock from './mock'
describe('<GridImage/>',()=>{
    test('shold render grid content',()=>{
     const {container} =  renderTheme(<GridImage {...mock}/>);
        expect(container).toMatchSnapshot()
    });
    
 });