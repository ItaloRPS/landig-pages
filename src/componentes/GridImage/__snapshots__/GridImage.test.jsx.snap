// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GridImage/> shold render grid content 1`] = `
.c2 {
  display: grid;
  grid-template-columns: repeat(auto-fill,minmax(280px,1fr));
  gap: 3.2rem;
}

.c3 {
  overflow: hidden;
}

.c4 {
  width: 100%;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c4:hover {
  -webkit-transform: scale(1.2) rotate(10deg);
  -ms-transform: scale(1.2) rotate(10deg);
  transform: scale(1.2) rotate(10deg);
}

.c0 {
  background: #FFFFFF;
  color: #0A1128;
  min-height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  padding: 5rem;
}

.c1 {
  font-size: 6.4rem;
  text-transform: uppercase;
}

@media (max-width:768px) {

}

<div>
  <div
    class="c0"
  >
    <div
      class=""
    >
      <h1
        class="c1"
      >
        My grid
      </h1>
      <div
        class="c2"
      >
        <div
          class="c3"
        >
          <img
            alt="Teste 1"
            class="c4"
            src="https://source.unsplash.com/random/800x800?r=1"
          />
        </div>
        <div
          class="c3"
        >
          <img
            alt="Teste 2"
            class="c4"
            src="https://source.unsplash.com/random/800x800?r=2"
          />
        </div>
        <div
          class="c3"
        >
          <img
            alt="Teste 3"
            class="c4"
            src="https://source.unsplash.com/random/800x800?r=3"
          />
        </div>
        <div
          class="c3"
        >
          <img
            alt="Teste 4"
            class="c4"
            src="https://source.unsplash.com/random/800x800?r=4"
          />
        </div>
        <div
          class="c3"
        >
          <img
            alt="Teste 5"
            class="c4"
            src="https://source.unsplash.com/random/800x800?r=5"
          />
        </div>
        <div
          class="c3"
        >
          <img
            alt="Teste 6"
            class="c4"
            src="https://source.unsplash.com/random/800x800?r=6"
          />
        </div>
      </div>
    </div>
    u
  </div>
</div>
`;
